src/components/Toast/ToastNotice.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { noop } from "src/utils";
+import { click, render } from "src/utils/rtl";
+import { ToastNotice } from "./ToastNotice";
+
+describe("ToastNotice", () => {
+  it("renders the default Toast notice layout", async () => {
+    // Given a Toast Notice component
+    const r = await render(<ToastNotice type={"error"} message={"Error"} onClose={noop} />);
+    // Then expect the default properties to exist in the layout
+    expect(r.getAllByTestId("toast_type")[0]).toBeInTheDocument();
```

Comment:
Suggestion: I think `r.toast_type_0` is a "cute" syntax that would be the same thing as this line?

Although, do we expect there to be `N` `toast_type`s? We could probably just do:

```
expect(r.toast_type()).toBeInTheDocument();
```

?

Reply:
I like that, although these tests cases are weird in that the `render` method was rendering the component twice.. I did some debugging and could not understand why it was rendering twice when I had separate `it` blocks and I am only calling the component once.. I also could not verify it was rendering twice in my stories either.. I will check with Brandon on this too